blog/2025-12-19-2025-end-of-year.md

```diff
@@ -0,0 +1,134 @@
+---
+authors: ["maya-singh"]
+title: "Inspektor Gadget in 2025: Making Low Level Observability Accessible to All"
+description: "Our 2025 year in review of the Inspektor Gadget project"
+draft: false
+tags: ["inspektor-gadget", "ebpf", "troubleshooting", "security"]
+slug: /2025/12/2025-year-in-review
+image: /media/splash/2025-12-19-end-of-year.jpg
+---
+
+# Inspektor Gadget 2025: Year in Review
+
+2025 marked a year of significant progress and innovation for the Inspektor Gadget project. Across observability, troubleshooting, and security, we focused relentlessly on the problems operators and developers face every day: reducing time to resolution when things go wrong, making powerful kernel-level visibility accessible without heroics, and improving trust in what runs in your clusters.
+
+This blog post is by no means comprehensive, but acts as a look back at the user problems we tackled this year, why we pursued solving them, and how users put the project to work.
+
+<!-- truncate -->
+
+## From Built-In to Image-Based Gadgets: The Big Shift
+
+We completed the [transition away from built-in gadgets](/blog/2025/02/deprecation-built-in-gadgets) to a fully modular, image-based architecture.
+
+Why it matters: users asked for flexibility, maintainability, and a path to custom gadgets without forking the core. Image-based gadgets deliver all three and lay the groundwork for an ecosystem that can flourish and grow beyond the confines of the project itself.
+
+- Gadgets ship as easily sharable standalone OCI images with metadata, eBPF code, and optional WASM modules.
+- Users can store and distribute gadgets in standard registries including [Artifact Hub](https://artifacthub.io/packages/search?kind=22), pin versions, and apply policies per image.
+- The architecture paves the way for innovation: community built gadgets, post-processing via WASM, and operators for a variety of export modes.
+
+## Enabling Continuous Observability: Inspektor Gadget + OpenTelemetry (OTel)
+
+We know that a key piece of incident prevention is effective monitoring and alerting. While many initially saw Inspektor Gadget as strictly a real time troubleshooting tool, we wanted to make it easier and more accessible to run IG continuously so users could be proactive around incident prevention. After discussion with the community, we learned that OpenTelemetry standards were the way to enable this. In 2025, we made our OTel integration more natural and configurable:
+
+- We added native OTel operators and improvements so gadgets can emit metrics and logs in standardized formats right out of the box.
```

Comment:
Link to https://inspektor-gadget.io/docs/latest/reference/export-logs and https://inspektor-gadget.io/docs/latest/reference/export-metrics#otlp-grpc?